ReactSkia/components/RSkComponentActivityIndicator.cpp

```diff
@@ -0,0 +1,26 @@
+#include <ReactSkia/activityindicator/react/renderer/components/activityindicator/ActivityIndicatorProps.h>
+#include "ReactSkia/components/RSkComponentActivityIndicator.h"
+
+#include "ReactSkia/utils/RnsLog.h"
+
+namespace facebook {
+namespace react {
+
+RSkComponentActivityIndicator::RSkComponentActivityIndicator(const ShadowView &shadowView)
+    : RSkComponent(shadowView) {}
+
+RSkComponentActivityIndicator::~RSkComponentActivityIndicator(){}
+
+RnsShell::LayerInvalidateMask  RSkComponentActivityIndicator::updateComponentProps(const ShadowView &newShadowView,bool forceUpdate) {return RnsShell::LayerInvalidateNone;}
```

Comment:
1. return to be done in seperate line.
2. one line space before new function start

Reply:
addressed